docs/development.rst

```diff
@@ -297,7 +297,10 @@ Checklist:
 - update ``CHANGES.rst``, based on ``git log $PREVIOUS_RELEASE..``
 - check version number of upcoming release in ``CHANGES.rst``
 - verify that ``MANIFEST.in`` and ``setup.py`` are complete
-- ``python setup.py build_api ; python setup.py build_usage ; python setup.py build_man`` and commit
+- ``python setup.py build_api ; python setup.py build_usage ; python
+  setup.py build_man`` and commit (be sure to build with Python 3.4 as
+  Python 3.6 added `more guaranteed hashing algorithms
```

Comment:
hmm, just noticed: it recommends 3.4 because 3.6 added.... - so, what about 3.5?